examples/facedetection/main.go

```diff
@@ -56,22 +55,22 @@ func main() {
 		fmt.Printf("Connection State has changed %s \n", connectionState.String())
 	})
 
-	md := mediadevices.NewMediaDevices(peerConnection)
-
 	vp8Params, err := vpx.NewVP8Params()
 	if err != nil {
 		panic(err)
 	}
 	vp8Params.BitRate = 100000 // 100kbps
 
+	md := mediadevices.NewMediaDevices(
+		peerConnection,
+		mediadevices.WithVideoEncoders(&vp8Params),
+		mediadevices.WithVideoTransformers(markFacesTransformer),
```

Comment:
I think Transforms are not `MediaDevices` wide setting, but are media specific one.
For example, user may have multiple cameras and may want to apply transform to one of them.

Reply:
yea... that's true. Oh well, let me redesign it again.